src/session/session_api.c

```diff
@@ -1651,12 +1651,14 @@ __session_commit_transaction(WT_SESSION *wt_session, const char *config)
         ret = EINVAL;
     }
 
+    F_SET(session, WT_SESSION_RESOLVE);
```

Comment:
I'd prefer `WT_SESSION_RESOLVING_TXN` to `WT_SESSION_RESOLVE`

Reply:
I agree, that's a better name.

Pushed in e23e16482.